enzy_htp/structure/structure_translator/rosetta_translator.py

```diff
@@ -0,0 +1,100 @@
+"""Instantiation of TranslatorBase for the Rosetta molecular modelling suite. This class 
+enables support for translation of canonical amino acid names to and from the standard
+AmberMD scheme in EnzyHTP.
+
+Author: Chris Jurich <chris.jurich@vanderbilt.edu>
+Date: 2024-05-27
+"""
+
+from plum import dispatch
+
+from ..atom import Atom
+from ..residue import Residue
+from ..structure import Structure
+
+
+from .translator_base import TranslatorBase
+
+
+class RosettaTranslator(TranslatorBase):
+    """Instantiation of TranslatorBase() for the Rosetta molecular modellig suite. """
```

Comment:
I think people call this concrete class more often (i.e., concrete class of TranslatorBase())

Reply:
You are right. The word I was thinking of was specialization. I have update the documentation to reflect that.